.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
```

Comment:
Are we sure we need to build for all these different versions?

Reply:
We probably don't *need* them, but they seem to cost us fairly little (except ~2 minutes of build time).  I say we leave them in, at least for now.

Note that I'm not sure being compatible with python 2.7 is super important, but we might as well leave it as long as we can.